src/panel/context/Explorer/index.tsx

```diff
@@ -30,17 +30,23 @@ export const ExplorerProvider: FC = ({ children }) => {
   >(undefined);
 
   useEffect(() => {
-    return addMessageHandler((o: DevtoolsExchangeOutgoingMessage) => {
-      if (o.type === "disconnect") {
+    return addMessageHandler((e: DevtoolsExchangeOutgoingMessage) => {
```

Comment:
Think might help differentiate _chrome messages_ from _debug events_

```suggestion
    return addMessageHandler((message: DevtoolsExchangeOutgoingMessage) => {
```